types/index.d.ts

```diff
@@ -63,7 +63,7 @@ declare class ShopifyToken {
    *     additional data
    * @public
    */
-  getAccessToken(shop: string, code: string): Promise<object>;
+  getAccessToken(shop: string, code: string): Promise<any>;
```

Comment:
Why this change?

Reply:
Ah ok, I didn't read commit message.

Reply:
Please wrap commit message body to 72 chars per line.

Reply:
Can we use `Record<string, string>` instead of `any`?

Reply:
yup